ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -157,6 +160,7 @@ private KsqlRestApplication(
     final String ksqlInstallDir = config.getString(KsqlRestConfig.INSTALL_DIR_CONFIG);
     this.uiFolder = isUiEnabled ? ksqlInstallDir + "/ui" : null;
     this.isUiEnabled = isUiEnabled;
+    this.mapper = getJsonMapper();
```

Comment:
I  think you missed my point.

- There are 8 production uses of `new ObjectMapper` and 3 calls to `getJsonMapper`. 
- You've added 3 calls to `.registerModule(new StructSerializationModule())`.
- We have other things that register with the mapper, e.g. `SchemaMapper` and `dk8Module`.
- The code to configure mappers is duplicated, (but not consistently), through the code base.

What I meant in the latest review is that it might be worth having a single ObjectMapper instance in the application, initialised and configured once and shared. This is preferable to doing the same configuration on different instances scattered through the code. e.g. 

```
public enum JsonMapper {
  INSTANCE;

  public final ObjectMapper mapper = new ObjectMapper();

  JsonMapper() {
    new SchemaMapper().registerToObjectMapper(mapper);
    mapper.registerModule(new Jdk8Module());
    mapper.registerModule(new StructSerializationModule());
  }
}
```

Of course, there are legitimate reasons to having different `ObjectMapper` instances with different configuration, e.g. for different use-cases. What I'm suggesting is, for one use-case there should only be one `ObjectMapper`.  Not several and code duplicated across the place.